src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,152 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
```

Comment:
Avoid starting class or method implementations with an empty line. Remove redundant empty lines as per the checklist: "Don't begin class or method implementation with an empty line."